app/_hub/kong-inc/key-auth/_index.md

```diff
@@ -384,6 +392,19 @@ Response:
 [consumer-object]: /gateway/latest/admin-api/#consumer-object
 [acl-associating]: /plugins/acl/#associating-consumers
 
+
+### Behavior matrix
+
+The following table describes how {{site.base_gateway}} behaves in various scenarios:
+
+Request                        | Proxied to upstream service? | Response status code
```

Comment:
@jschmid1 For the entries in this column, does "has valid apikey" display exactly like that in the request? Or is this more of a description of what happens in the request? Ex. "The request has a valid API key".

Reply:
This is more of a description. Feel free to adapt it